.ci/releaseBuild.yml

```diff
@@ -106,10 +106,16 @@ stages:
         Write-Host "local repo is: " + $vstsCommandString
         Write-Host "##$vstsCommandString"
 
-        Register-PSRepository -Name local -SourceLocation $(localRepo)
+        Register-PSRepository -Name local -SourceLocation $localRepo
         Publish-Module -Path $(signOutPath) -Repository local
       displayName: Publish module to local repository
 
+    # finally publish the parts of the build which will be used in the next stages
+    # if it's not published, the subsequent stages will not be able to access it.
+    - publish: "$(Build.SourcesDirectory)/OSS_Microsoft_CompatPowerShellGet"
+      artifact: build
+      displayName: publish build directory

```

Comment:
We can keep this if we want to. But we can remove the download in the compliance stage. We dont use the downloaded artifact.

Reply:
Can I remove the following:
```
    - download: current
      artifact: build

```